apps/accounts/models.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
+from django.db import models
+from django.utils import timezone
+
+from ckeditor.fields import RichTextField
+from sorl.thumbnail import ImageField
+
+from .managers import UserManager
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+    email = models.EmailField('email address', unique=True)
```

Comment:
Shall we switch this to the case insensitive text field from postgres? Causes fewer problems later!

Reply:
+1